code/datums/components/electrified_buckle.dm

```diff
@@ -1,26 +1,46 @@
-#define ELECTRIC_BUCKLE_WAIT_TIME 5 SECONDS
 ///divide the power in the cable net under parent by this to determine the shock damage
-#define ELECTRIC_BUCKLE_SHOCK_STRENGTH_DIVISOR  3000
+#define ELECTRIC_BUCKLE_SHOCK_STRENGTH_DIVISOR 5000
 ///it will not shock the mob buckled to parent if its required to use a cable to shock and the cable has less than this power availaible
 #define ELECTRIC_BUCKLE_MINUMUM_POWERNET_STRENGTH 10
 
+
+/*	electrified_buckle component:
```

Comment:
This isn't the proper syntax, check AUTODOC_GUIDE.md.

Reply:
Here and everywhere else.

Reply:
i think i fixed it